content/docs/tina-cloud/beta-migration.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Upgrading from Alpha to Beta
+---
+
+## Beta Changes
+
+Aside from some user interface improvements in the Tina Cloud Dashboard, we've also been making a bunch of changes
+under the hood. One of the biggest of these changes is how we handle user accounts.
+
+Previously when you created a Tina Cloud account you were creating an **organization** that you accessed at `https://OrgName.tina.io`. We thought that giving each organization its own unique subdomain and pool of users would be neat, but after spending some time in that paradigm it turned out to be a lot of added complexity that didn't add much value.
+
+With the release of the beta we're moving user accounts to the forefront. Whenever you create a Tina Cloud account you'll always access it at https://app.tina.io. This means that even
+if you are part of multiple organizations, you'll only have a single Tina account.
+
+> Tools for creating/managing organizations within Tina Cloud accounts are coming. For now we create a default organization, which you'll still be able to invite users to.
+
+## Migration Path
+
+Since we're changing how user accounts work within Tina Cloud, existing alpha users must **recreate their Tina Cloud account** and **update their Tina Cloud Site**. The good news is that since we don't control your content, there really isn't anything else to do.
+
+### Recreate your Tina Cloud Account
+
+- Head to https://app.tina.io and sign up for a new account.
+- Log in to your newly created account [recreate any existing **Apps**](/docs/tina-cloud/dashboard/#apps).
+- Invite any users to your newly created account.
+
+### Update your Tina Cloud site
+
+1. Remove the `tina-graphql-gateway` and `tina-graphql-gateway-cli` dependencies.
+
+```
+yarn remove tina-graphql-gateway-cli tina-graphql-gateway
+```
+
+2. Add the new Tina dependencies.
+
+```
+yarn add tinacms @tinacms/auth
+yarn add -D @tinacms/cli
+```
+
+3. Change any existing import from `tina-graphql-gateway` to `tinacms`.
+
+4. Update `package.json`. Replace `yarn tina-gql` calls with `yarn tinacms`.
+
+5. Update your site's environment variables.
+
+- Delete `NEXT_PUBLIC_ORGANIZATION_NAME`
```

Comment:
In addition to this, do we need to tell the user to remove the orgId from the Client creation?
https://github.com/tinacms/tinacms.org/commit/10d67f4d4caf0682c9d3221ea9a3178291136a9e#diff-732071cf7d2a0299edf199f91126d4343f052eee3948dae5c0f4f6408dc6c769L30

Maybe we add a bullet point to update the Client creation like so: https://tina.io/docs/tina-cloud/client/#the-tinacms-api-client

Reply:
@jamespohalloran Those sections are outdated, most of the new documentation is still in this PR https://github.com/tinacms/tinacms.org/pull/946

Reply:
Ok, so if I'm understanding, we'll still need to add this migration step, but we'll instead link to: https://github.com/tinacms/tinacms.org/pull/946/files#diff-f5b803b1d78f9603c9177621d55c03d2b5ed0371b268d70d9504a76539a48245

Reply:
They don't need to change any of that if they're already set up

Reply:
Ok I'm confused :) The Client API used to take an organizationId, but with v2 it doesn't need to. Shouldn't that change be apart of the migration doc if the user needs to remove that parameter? Or do I have something wrong?

Reply:
Oh I see, yeah they'll want to remove it, but what I was saying was that we don't need them to do all of that stuff since they already have the `Client` set up somewhere, we can just tell them they can remove it where they've referenced it.